Firmware/heatbed_pwm.cpp

```diff
@@ -137,12 +140,17 @@ ISR(TIMER0_OVF_vect)          // timer compare interrupt service routine
 	case States::ONE:             // state ONE - we'll either stay in ONE or change to FALL
 		OCR0B = 255;
 		slowCounter += slowInc;   // this does software timer_clk/256 or less
-		if( slowCounter < pwm ){
-			return;
+		if (!bedPWMDisabled){ //disable heating as soon as possible

```

Comment:
is it really correct to have the slowCounter outside of the "if" statement? It basically continues to tick the SW (aka slow) PWM. Wouldn't it be better to just keep the States::ONE when bed PWM is to be disabled?